index.js

```diff
@@ -0,0 +1,256 @@
+import nodePlugin from "eslint-plugin-n";
+import importPlugin from 'eslint-plugin-import';
+import promisePlugin from 'eslint-plugin-promise';
+import js from '@eslint/js';
+
+export default [
+    js.configs.recommended,
+
+    {
+        plugins: {
+            'n': nodePlugin,
+            'import': importPlugin,
+            'promise': promisePlugin,
+        },
+
+        rules: {
+            "indent": ["error", 4],
+
+            // revised rules from "standard"
```

Comment:
I took rules from "standard", and removed ones conflicting with eslint/recommended and prettier.
I also removed some (not all) rules deprecated in eslint
In complex situations, I preferred to remove eslint rules related to styling (e.g. dangling commas etc)